packages/web/components/discover/filter/discover-filter-from.component.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Form, Select, DatePicker, Slider, Button, Checkbox } from 'antd';
+import { FilterDiscoverySectionComponent } from './discover-filter-section.component';
+import {
+  GetDiscoverQueryVariables,
+  useGetLanguagesQuery,
+  useGetGenresQuery,
+} from '../../../utils/graphql';
+interface DiscoverFilterFormComponentProps {
+  params: GetDiscoverQueryVariables;
+  onFinish: (formParams: GetDiscoverQueryVariables) => void;
+}
+export function DiscoverFilterFormComponent(
+  props: DiscoverFilterFormComponentProps
+) {
+  const languagesQuery = useGetLanguagesQuery();
+  const genresQuery = useGetGenresQuery();
+
+  const TMDBLanguages = languagesQuery.data?.languages;
+  const TMDBMovieGenres = genresQuery.data?.genres.movieGenres;
+
+  const [form] = Form.useForm();
+  const { Option } = Select;
```

Comment:
Don't create a new variable `Option` at every render, just use `<Select.Option>` in the markup 👍